flocker/node/agents/cinder.py

```diff
@@ -311,28 +312,25 @@ def reached_desired_state(self):
         Raise an exception if a non-valid state is reached or if the
         desired state is not reached within the supplied time limit.
         """
-        # Could be more efficient.  FLOC-1831
-        for listed_volume in self.volume_manager.list():
-            if listed_volume.id == self.expected_volume.id:
-                # Could miss the expected status because race conditions.
-                # FLOC-1832
```

Comment:
It would be nice not to lose this comment.  As far as I can tell, it's still relevant.